vyper/compiler/output.py

```diff
@@ -206,7 +207,15 @@ def build_ir_runtime_dict_output(compiler_data: CompilerData) -> dict:
 
 
 def build_metadata_output(compiler_data: CompilerData) -> dict:
-    sigs = compiler_data.function_signatures
+    _ = compiler_data.function_signatures
```

Comment:
why do we need to compute function_signatures?

Reply:
cause the `_ir_info` is not set - the error is:
```
ret["_ir_identifier"] = func_t._ir_info.ir_identifier
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'ir_identifier'
```

on line https://github.com/vyperlang/vyper/blob/063f1900a1b2b32bf8d0f091027dd0fb16e99de8/vyper/compiler/output.py#L224

Reply:
add a comment -- `# need ir info to be computed`